src/modules/Jobs/components/applyModalQuestions/index.tsx

```diff
@@ -56,8 +56,8 @@ const ApplyModalQuestions: React.FC<ApplyModalQuestionsProps> = ({ answers, setA
               })}
               onChange={type => handleSelectOption(question.id, Number(type.value), type.label)}
               errors={
-                questionErrors.find(e => e.id === question.id)
-                  ? [questionErrors.find(e => e.id === question.id).message]
+                questionErrors.find(e => e.id === question.id)?.message
```

Comment:
no need to first line
`[questionErrors.find(e => e.id === question.id)?.message || '']` is enough:
1- if message found -> message
2- if message not found -> undefined || '' -> would be empty string

Reply:
line 71, 72 too

Reply:
The `errors` is an array or undefined. In this case, if message not found, [''] is returned, while we want an undefined